protocol-units/execution/maptos/opt-executor/src/background/transaction_pipe.rs

```diff
@@ -481,6 +450,52 @@ mod tests {
 		(context, transaction_pipe, tempdir)
 	}
 
+	#[derive(Clone)]
+	pub struct TxPipeTestDaSequencerClient {
+		pub received_tx: Arc<Mutex<Vec<MvTransaction>>>,
+	}
+
+	impl TxPipeTestDaSequencerClient {
+		fn new() -> Self {
+			TxPipeTestDaSequencerClient { received_tx: Arc::new(Mutex::new(vec![])) }
+		}
+	}
+
+	impl DaSequencerClient for TxPipeTestDaSequencerClient {
+		/// Stream reads from a given height.
+		async fn stream_read_from_height(
```

Comment:
I don't think you need the doc comments if they are already on the trait.

Reply:
Removed.